core/services/pipeline/task.mean.go

```diff
@@ -0,0 +1,70 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/shopspring/decimal"
+	"go.uber.org/multierr"
+)
+
+type MeanTask struct {
+	BaseTask      `mapstructure:",squash"`
+	Values        string `json:"values"`
+	AllowedFaults string `json:"allowedFaults"`
+	Precision     string `json:"precision"`
+}
+
+var _ Task = (*MeanTask)(nil)
+
+func (t *MeanTask) Type() TaskType {
+	return TaskTypeMean
+}
+
+func (t *MeanTask) Run(_ context.Context, vars Vars, _ JSONSerializable, inputs []Result) (result Result) {
+	var (
+		maybeAllowedFaults MaybeUint64Param
+		maybePrecision     MaybeUint64Param
```

Comment:
This is a Uint64 but it's cast to an int32 later, is this safe?

Reply:
Fixed